modules/mtl/shared/src/main/scala/retry/mtl/RetrySyntax/retry.scala

```diff
@@ -4,14 +4,7 @@ import cats.effect.Temporal
 import cats.mtl.Handle
 import retry.{RetryDetails, RetryPolicy}
 
-trait RetrySyntax:
-  implicit final def retrySyntaxMtlError[M[_], A](
-      action: => M[A]
-  ): RetryingMtlErrorOps[M, A] =
-    new RetryingMtlErrorOps[M, A](action)
-
-final class RetryingMtlErrorOps[M[_], A](action: => M[A]):
-
+extension [M[_], A](action: M[A])
```

Comment:
Is the file name correct?

Reply:
I beleive it should be `/syntax/RetrySyntax.scala`

Reply:
Yes, I think so too.
Done on: f50cc1d27e4d0d7220607d6ec6b5e31ffebc825d